.github/workflows/build-world.yaml

```diff
@@ -0,0 +1,96 @@
+name: Build the world
+
+on:
+  workflow_dispatch:
+    
+jobs:
+  build:
+    name: Build OS packages
+    runs-on:
+      - self-hosted
+      - ${{ matrix.arch }}
+
+    # Only run one build per arch at a time
+    concurrency:
+      group: build-world-${{ matrix.arch }}
+
+    strategy:
+      matrix:
+        arch: [ "x86_64", "arm64" ]
+      fail-fast: false
+
+    steps:
+      - name: Generate snapshot date
+        id: snapshot-date
+        run: |
+          echo "date=$(date -u +%Y%m%d)" >> $GITHUB_OUTPUT
+          echo "epoch=$(date -u +%s)" >> $GITHUB_OUTPUT
+        shell: bash
+
+      - uses: actions/checkout@v3
+
+      - run: echo "${{ secrets.MELANGE_RSA }}" > ./wolfi-signing.rsa
+
+      - run: |
+          sudo mkdir -p /etc/apk/keys
+          sudo cp ./wolfi-signing.rsa.pub /etc/apk/keys/wolfi-signing.rsa.pub
+
+      - uses: chainguard-dev/actions/setup-melange@main
+
+      - id: auth
+        name: 'Authenticate to Google Cloud'
+        uses: google-github-actions/auth@v0
+        with:
+          workload_identity_provider: "projects/618116202522/locations/global/workloadIdentityPools/prod-shared-e350/providers/prod-shared-gha"
+          service_account: "prod-images-ci@prod-images-c6e5.iam.gserviceaccount.com"
+
+      - uses: google-github-actions/setup-gcloud@v0
+        with:
+          project_id: prod-images-c6e5
+
+      - name: 'Check that GCloud is properly configured'
+        run: |
+          gcloud info
+          gcloud --quiet alpha storage ls
+
+      - name: 'Prepare package repository'
+        run: |
+          mkdir -p "${{ github.workspace }}/packages"
+          cp /etc/apk/keys/wolfi-signing.rsa.pub ${{ github.workspace }}/packages/wolfi-signing.rsa.pub
+
+      - name: "Install wolfictl onto PATH"
+        run: |
+          # Copy wolfictl out of the wolfictl image and onto PATH
+          TMP=$(mktemp -d)
+          docker run --rm -i -v $TMP:/out --entrypoint /bin/sh ghcr.io/wolfi-dev/sdk:latest@sha256:78751faebed64630b64351fff0431e38cf08c70997c19a72146698ced651aad0 -c "cp /usr/bin/wolfictl /out"
+          echo "$TMP" >> $GITHUB_PATH
+
+      - name: 'Build Wolfi'
+        run: |
+          # Run all and allow failures, all we really care about is the buildlogs for failures
+          make MELANGE="sudo -E melange" MELANGE_DIR=/usr/share/melange KEY=wolfi-signing.rsa BUILDWORLD=no REPO="${{ github.workspace }}/packages" -j1 -k all
```

Comment:
keeping this `BUILDWORLD=no` intentionally since the goal for now is to get through a build of _all_ our packages, and `BUILDWORLD=no` + `-k` won't get us very far if we can't rely on prior state